docs/source/concepts/runner.rst

```diff
@@ -114,11 +114,13 @@ Send a test request:
     {"is_positive":true}%
 
 
-The :code:`SUPPORT_NVIDIA_GPU` and :code:`SUPPORT_CPU_MULTI_THREADING` class attribute indicates
-whether or not this Runnable class implementation supports using GPU or utilizes multi
-threading. Since NLTK library doesn't support utilizing GPU nor multi-cpu-core natively,
-they are both set to False. This information is used by the BentoServer scheduler
-to determine the worker pool size of this runner.
+The :code:`supported_resources` class parameter indicates which resources this Runnable class
+implementation supports. The only currently pre-defined resource is ``"nvidia.com/gpu"``.
+:code:`supports_multi_threading` indicates whether or not the runner supports CPU multi-threading.
+Since the NLTK library doesn't support utilizing GPU or multiple CPU cores natively, no resources
+are specified, and ``supports_multi_threading`` is set to False. This is the default configuration.
+This information is used by the BentoServer scheduler to determine the worker pool size of this
+runner.
```

Comment:
```suggestion
The :code:`supported_resources` class parameter indicates which resources this Runnable class
implementation supports. The only currently pre-defined resource is ``"nvidia.com/gpu"``.

The :code:`supports_multi_threading` class parameter indicates whether or not the runner supports CPU multi-threading.

Since the NLTK library doesn't support utilizing GPU or multiple CPU cores natively, no resources
are specified, and ``supports_multi_threading`` is set to False. This is the default configuration.
This information is used by the BentoServer scheduler to determine the worker pool size of this
runner.
```